### PR TITLE
[Components with more than 1 service] LPS-183006, LPS-183634 Refactor BulkSelectionFactory and BulkSelectionAction components

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-183006](https://issues.liferay.com/browse/LPS-183006),
[LPS-183634](https://issues.liferay.com/browse/LPS-183634)

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:document

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:dl-store

**Comment by zhangyan0701** [comment]
ci:test:document

**Comment by catarineaguiar** [comment]
@RenanUmV rebase and re-run test suites, please.

Thanks!

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
liferay/ci:test:document

**Comment by RenanUmV** [comment]
liferay/ci:test:dl-store

**Comment by RenanUmV** [comment]
liferay/ci:test:document

**Comment by RenanUmV** [comment]
ci:test:dl-store

**Comment by zhangyan0701** [comment]
Semantic versioning issue:
```
     [exec] > Task :apps:document-library:document-library-api:baseline
     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec]   com.liferay.document.library.bulk.selection        REMOVED    1.0.0      1.0.0      -          PACKAGE REMOVED, UNNECESSARY PACKAGEINFO
     [exec] [Baseline Warning] Bundle Version Change Recommended: 21.0.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.document.library.api-20.0.1.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.document.library.api/20.0.0/b9ed71678d249e4638a392879310e785faefdbb5/com.liferay.document.library.api-20.0.0.jar
     [exec] 
```
Full console log:
https://testray.liferay.com/reports/production/logs/2023-05/test-1-32/test-portal-acceptance-pullrequest(master)/9542/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0

File: modules/apps/bulk/bulk-selection-test/src/testIntegration/java/com/liferay/bulk/selection/test/util/TestBulkSelectionFactory.java [hunk 1]
```diff
@@ -31,9 +31,7 @@
 /**
  * @author Alejandro Tardín
  */
-@Component(
-	service = BulkSelectionFactory.class
-)
+@Component(service = BulkSelectionFactory.class)
```
Comment (rafaprax) [inline modules/apps/bulk/bulk-selection-test/src/testIntegration/java/com/liferay/bulk/selection/test/util/TestBulkSelectionFactory.java:34]: Hey @RenanUmV, this change must be into this commit https://github.com/liferay-core-infra/liferay-portal/pull/4522/commits/b7cd433b95ea4b949161a4cf94f91c8a8792493a

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:dl-store

**Comment by RenanUmV** [comment]
ci:test:document

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:document

**Comment by RenanUmV** [comment]
ci:test:dl-store

**Comment by catarineaguiar** [comment]
no regressions found.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Pull request closed without merging**